rich/traceback.py

```diff
@@ -41,6 +53,7 @@ def install(
     theme: Optional[str] = None,
     word_wrap: bool = False,
     show_locals: bool = False,
+    exclude_locals: Optional[Tuple[str]] = None,
```

Comment:
The typing could probably be relaxed a little. I usually go for `Iterable` so the caller could pass in any container with strings.